db/migrate/20170417192246_create_interactions_publications_table.rb

```diff
@@ -0,0 +1,15 @@
+class CreateInteractionsPublicationsTable < ActiveRecord::Migration
+	def up
```

Comment:
The indentation in this file is off (too far indented). The standard is two spaces per indentation level.